manim/utils/tex_file_writing.py

```diff
@@ -197,7 +197,7 @@ def convert_to_svg(dvi_file, extension, page=1):
             "dvisvgm",
             "--pdf" if extension == ".pdf" else "",
             "-p " + str(page),
-            '"{}"'.format(dvi_file),
+            f'"{dvi_file}"',
```

Comment:
same comment as above : why '"x"'?

Reply:
The file name will be set between double quotes for the shell, so I think it makes sense to have the outer quotes being single quotes

Reply:
alright. I just found a source that this is possible to do it like this in python:
If either single or double quote is a part of the string itself, then the string must be placed in double or single quotes respectively.
https://www.tutorialspoint.com/What-is-the-difference-between-single-and-double-quotes-in-python#:~:text=What%20is%20the%20difference%20between%20single%20and%20double%20quotes%20in%20python%3F,-PythonServer%20Side&text=In%20Python%2C%20such%20sequence%20of,representations%20can%20be%20used%20interchangeably.